src/app/plugins/ta.plugin.ts

```diff
@@ -71,7 +71,9 @@ export class TaPlugin extends Plugin {
       .array()
       .map((ta) => ta.user.toString()) //Convert to pingable mentions
       .join(' ');
-    message.channel.send(`${message.author} asks: \n> ${question}\n${mentions}`);
+    message.channel.send(`${message.author} asks: \n>>> ${question}\n${mentions}`, {
+      split: { prepend: '>>>' },
+    });
```

Comment:
Did you test this?

Reply:
All have been tested

Reply:
Pretty sure this puts the mentions in a quote too

Reply:
It does, but it looks worse if the message comes out of the quotes than with the mention inside the quotes.

Reply:
I'll post images of results in discord in a second.